conf/logrotate.conf

```diff
@@ -20,6 +20,23 @@ notifempty
   endscript
 }
 
+## AUTH
+/var/log/st2/st2auth.log {
```

Comment:
Should we break out results tracker, notifier, and sensor container into separate entries for audit and non-audit to be consistent? They are currently using *.log. Last question. :-)